toolkit/tools/pkgworker/pkgworker.go

```diff
@@ -41,6 +42,7 @@ var (
 	repoFile             = app.Flag("repo-file", "Full path to local.repo").Required().ExistingFile()
 	rpmsDirPath          = app.Flag("rpm-dir", "The directory to use as the local repo and to submit RPM packages to").Required().ExistingDir()
 	srpmsDirPath         = app.Flag("srpm-dir", "The output directory for source RPM packages").Required().String()
+	toolchainDirPath     = app.Flag("toolchain-rpm-dir", "Directory that contains already built toolchain RPMs. Should contain top level directories for architecture.").Required().ExistingDir()
```

Comment:
nit: consider changing to "directories for architectures" or "directories per architecture".

Reply:
New: "Directory that contains already built toolchain RPMs. Should contain a top level directory for each architecture"